server/controllers/guess.js

```diff
@@ -0,0 +1,24 @@
+const connection = require('../config/db.js');
+
+const insert_coords = (req,res)=>{
+    sql = "INSERT INTO guess_attempt (locationLatGuess,locationLongGuess) VALUES (?,?)";
+    console.log("got here in guess.js")
```

Comment:
Probably want to remove this and the logging of the coordinates for the actual website (fine to have for your local testing)